### PR TITLE
update SD to v6.1.0

### PR DESCRIPTION
**Pull request by hathach**
increase SD to 6.1.0

### CODE REVIEW

**Comment by dhalbert** [comment]
You were having trouble with 6.1.0, right? Is that problem gone now?

**Comment by dhalbert** [comment]
Oops, now I see the answer in https://github.com/adafruit/Adafruit_nRF52_Bootloader/issues/20

**Comment by hathach** [comment]
@dhalbert yeah Nordic support team help to troubleshoot it, here is the detail
https://devzone.nordicsemi.com/f/nordic-q-a/37473/advertising-does-not-start-with-s140-6-1-0-same-hex-run-without-issue-with-s140-6-0-0

File: ports/nrf/drivers/bluetooth/download_ble_stack.sh [hunk 1]
```diff
@@ -35,6 +35,22 @@ function download_s132_nrf52_5_0_0
     cd -
 }
 
+function download_s132_nrf52_6_1_0
```
Comment (dhalbert) [inline ports/nrf/drivers/bluetooth/download_ble_stack.sh]: Not that important, but all these download functions could be refactored into one that takes a few arguments.

Comment (dhalbert) [reply ports/nrf/drivers/bluetooth/download_ble_stack.sh]: Maybe next time.

**Comment by dhalbert** [comment]
pca10056 build is failing in travis nrf52* builds are OK.

File: ports/nrf/drivers/bluetooth/download_ble_stack.sh (hunk 1 above)
Comment (hathach) [reply ports/nrf/drivers/bluetooth/download_ble_stack.sh]: yeah, I also feel that. I am not a fan of shell script as well. It is better to have some kind of dictionary for SD name and link :+1:

Comment (dhalbert) [reply ports/nrf/drivers/bluetooth/download_ble_stack.sh]: You could rewrite it as a Python script. :smiley:

**Comment by hathach** [comment]
Sorry, forgot to add new SD it to the download all (no parameters). Only test it when already download the SD. Travis is very handy :)

File: ports/nrf/drivers/bluetooth/download_ble_stack.sh (hunk 1 above)
Comment (hathach) [reply ports/nrf/drivers/bluetooth/download_ble_stack.sh]: That will be a very awful python script. I am way too bad with python :(

**Comment by dhalbert** [comment]
>PS: feather52832 does not fail since I leave it as it is for arturo182 to decide later.

Is there any reason to hold back to 6.0.0 on feather 52832 and feather 52840? Is the base code the same (the fix you added?)

**Comment by hathach** [comment]
No, the only reason is probably the flash size. In Arduino repo, I will also drop support for SD v2 as well for the next nrf52840 release. It is the pain to have ble driver to support 2 flows of the APIs. Since @arturo182  ask to wait for him to come back. And he is the one writing the ble driver ( I haven't touched it so far). I guess let's wait for him to clean them up later.

**Comment by dhalbert** [comment]
Can we wait for him to return before I merge this PR, then, so we won't have two versions in the tree?

**Comment by hathach** [comment]
ah, I sorry, I misunderstood your question, since you askabout nrf52832, nrf52832 does not have SD 6.0.0 support. only v2 and v5 
https://github.com/adafruit/circuitpython/tree/master/ports/nrf/boards/feather52832#s132-v201-single-bank-recommended

For 6.0.0, only nrf52840 is supported, 6.1.0 is binary compatible with 6.0.0 the flash size is the same, there is no reason to hold 6.0.0. I will remove them in the next push if you like

**Comment by hathach** [comment]
Sorry for confusion, just remove s140 6.0.0 and s132 6.1.0 ( not supported yet )

**Comment by hathach** [comment]
@dhalbert 6.0.0 is removed, it should be safe to merge now

**Pull request merged**